src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderIsCreatedIfStartTimeIsInThePastAndEndTimeIsInTheFuture.java

```diff
@@ -0,0 +1,65 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC16: Verify an Out Of Order can be created when the start date is in the past 
+ * but the end date is in the future
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderIsCreatedIfStartTimeIsInThePastAndEndTimeIsInTheFuture {
+
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(9).get("Room Name");
```

Comment:
As well here, keep only roomName and title as global variables, others declare inside the test method